environment/service.go

```diff
@@ -83,30 +83,55 @@ type Service struct {
 	templatesRepoDir  string
 }
 
-func NewService(templatesRepo, templatesRepoBlob, templatesRepoDir string) *Service {
-	return &Service{
-		templatesRepo:     templatesRepo,
-		templatesRepoBlob: templatesRepoBlob,
-		templatesRepoDir:  templatesRepoDir,
+func NewService() *Service {
+	return &Service{}
+}
+
+func NewServiceForUserData(user *authclient.UserDataAttributes) *Service {
+	service := NewService()
+	if user != nil {
+		if user.FeatureLevel != nil && *user.FeatureLevel != "internal" {
```

Comment:
we should have a constant for this `internal` feature level ;)

Reply:
done c60a726